pkg/imagestore/imagestore.go

```diff
@@ -108,8 +116,9 @@ func validateVersions(versions []map[string]string) error {
 	return nil
 }
 
-func downloadURLToFile(url string, path string) error {
-	resp, err := http.Get(url)
+func downloadURLToFile(httpClient *http.Client, url string, path string) error {
```

Comment:
Is there any reason not to make this a method on `rhcosStore` rather than passing in the client?